pkg/services/librarypanel/librarypanel.go

```diff
@@ -0,0 +1,61 @@
+package librarypanel
+
+import (
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+// LibraryPanel is the service the Panel Library feature.
+type LibraryPanel struct {
```

Comment:
LibraryPanelService? (to not confuse it with the entity)

Reply:
or PanelLibraryService

Reply:
I just pushed that change :)

Reply:
lol, was thinking dam that was fast!